planner/core/preprocess.go

```diff
@@ -15,6 +15,7 @@ package core
 
 import (
 	"fmt"
+	"github.com/pingcap/tidb/meta/autoid"
```

Comment:
Please update the order of imported package.

Reply:
Updated. PTAL :)